lib/config.go

```diff
@@ -168,6 +168,10 @@ type RuntimeConfig struct {
 	// ContainerExitsDir is the directory in which container exit files are
 	// written to by conmon.
 	ContainerExitsDir string `toml:"container_exits_dir"`
+
+	// AllowStrace determinates whether the cri-o strace annotation takes effect
+	// or not.
```

Comment:
I think the “not” case could use more specifics.  Perhaps “or not” → “or is silently ignored”?  We'll also want similar text in a `docs/crio.8.md` entry.